Panic in dBFT during single node startup

I ran single node instance at latest master (0.90.0-pre-587-g8ad355b4) and it threw panic.

```
2020-06-29T13:55:05.728+0300    INFO    no storage version found! creating genesis block
2020-06-29T13:55:05.729+0300    INFO    service hasn't started since it's disabled      {"service": "Prometheus"}
2020-06-29T13:55:05.729+0300    INFO    service hasn't started since it's disabled      {"service": "Pprof"}

    _   ____________        __________
   / | / / ____/ __ \      / ____/ __ \
  /  |/ / __/ / / / /_____/ / __/ / / /
 / /|  / /___/ /_/ /_____/ /_/ / /_/ /
/_/ |_/_____/\____/      \____/\____/

/NEO-GO:0.90.0-pre-587-g8ad355b4/

2020-06-29T13:55:05.729+0300    INFO    starting rpc-server     {"endpoint": ":30333"}
2020-06-29T13:55:05.729+0300    INFO    node started    {"blockHeight": 0, "headerHeight": 0}
2020-06-29T13:55:05.729+0300    INFO    node reached synchronized state, starting consensus
2020-06-29T13:55:06.059+0300    INFO    initializing dbft       {"height": 1, "view": 0, "index": 0, "role": "Primary"}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x1 pc=0xa7a76d]

goroutine 58 [running]:
github.com/nspcc-dev/neo-go/pkg/consensus.(*Payload).SetViewNumber(0xc0001ae000, 0x24ee298500000000)
        /home/alexvanin/Project/Neo/neo-go/pkg/consensus/payload.go:61 +0xd
github.com/nspcc-dev/dbft.(*Context).withPayload(0xc00009d8c0, 0x14e3120, 0xce81e0, 0xc0002983c0, 0xc00010fe70, 0xa69f08)
        /home/alexvanin/Project/Neo/neo-go/vendor/github.com/nspcc-dev/dbft/send.go:179 +0xef
github.com/nspcc-dev/dbft.(*Context).makePrepareRequest(0xc00009d8c0, 0x5, 0xb)
        /home/alexvanin/Project/Neo/neo-go/vendor/github.com/nspcc-dev/dbft/send.go:27 +0x145
github.com/nspcc-dev/dbft.(*DBFT).sendPrepareRequest(0xc00009d8c0)
        /home/alexvanin/Project/Neo/neo-go/vendor/github.com/nspcc-dev/dbft/send.go:31 +0x45
github.com/nspcc-dev/dbft.(*DBFT).start(0xc00009d8c0)
        /home/alexvanin/Project/Neo/neo-go/vendor/github.com/nspcc-dev/dbft/dbft.go:268 +0x26f
github.com/nspcc-dev/dbft.(*DBFT).Start(0xc00009d8c0)
        /home/alexvanin/Project/Neo/neo-go/vendor/github.com/nspcc-dev/dbft/dbft.go:82 +0x59
github.com/nspcc-dev/neo-go/pkg/consensus.(*service).Start(0xc0000a55e0)
        /home/alexvanin/Project/Neo/neo-go/pkg/consensus/consensus.go:179 +0x56
github.com/nspcc-dev/neo-go/pkg/network.(*Server).tryStartConsensus(0xc0000a7b80)
        /home/alexvanin/Project/Neo/neo-go/pkg/network/server.go:308 +0xda
github.com/nspcc-dev/neo-go/pkg/network.(*Server).Start(0xc0000a7b80, 0xc0000398c0)
        /home/alexvanin/Project/Neo/neo-go/pkg/network/server.go:173 +0x202
created by github.com/nspcc-dev/neo-go/cli/server.startServer
        /home/alexvanin/Project/Neo/neo-go/cli/server/server.go:358 +0x480
```

My config:

```
ProtocolConfiguration:
  Magic: 56753
  SecondsPerBlock: 10
  MemPoolSize: 50000
  StandbyValidators:
    - 02b3622bf4017bdfe317c58aed5f4c753f206b7db896046fa7d774bbc4bf7f8dc2
  SeedList:
    - 172.200.0.1:20333
  VerifyBlocks: true
  VerifyTransactions: true

ApplicationConfiguration:
  DBConfiguration:
    Type: "boltdb" 
    BoltDBOptions:
      FilePath: "./db/privnet.bolt"
  #  Uncomment in order to set up custom address for node.
  #  Address: 127.0.0.1
  NodePort: 20333
  Relay: true
  DialTimeout: 3
  ProtoTickInterval: 2
  PingInterval: 30
  PingTimeout: 90
  MaxPeers: 10
  AttemptConnPeers: 5
  MinPeers: 0
  RPC:
    Enabled: true
    EnableCORSWorkaround: false
    Port: 30333
  Prometheus:
    Enabled: false
    Port: 20001
  Pprof:
    Enabled: false
    Port: 20011
  UnlockWallet:
    Path: "./wallets/node-wallet.json"
    Password: "one"
```

--- a/pkg/consensus/consensus.go
+++ b/pkg/consensus/consensus.go
@@ -167,6 +167,7 @@ var (
 func NewPayload(m netmode.Magic) *Payload {
 	return &Payload{
 		network: m,
+		message: new(message),
 	}
 }
 

--- a/pkg/consensus/payload.go
+++ b/pkg/consensus/payload.go
@@ -171,7 +171,7 @@ func (p *Payload) EncodeBinaryUnsigned(w *io.BinWriter) {
 	w.WriteU32LE(p.height)
 	w.WriteU16LE(p.validatorIndex)
 
-	if p.message != nil {
+	if p.data == nil {
 		ww := io.NewBufBinWriter()
 		p.message.EncodeBinary(ww.BinWriter)
 		p.data = ww.Bytes()

